analyzer-tests/src/test/java/ffb/analyzer/tests/application/ApplicationInitializationTests.java

```diff
@@ -0,0 +1,15 @@
+package ffb.analyzer.tests.application;
+
+import ffb.analyzer.application.ApplicationInitializer;
+import org.junit.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(classes = ffb.analyzer.application.ApplicationInitializer.class)
+public class ApplicationInitializationTests {
+
+    @Test
+    public void contextLoads () {
+        ApplicationInitializer.main(new String[1]);
```

Comment:
Should we have an assertion here that checks for the "Application has started." detail in the log file, since that should always execute on startup?

Reply:
I actually just removed the `ApplicationRunner` interface. If we need to hook into the startup, we can add it back. Right now, this is just a way for us to quickly check that the application loads correctly.